src/server/api/routers/questions/delete-chunk.ts

```
@@ -14,274 +12,189 @@ import {
 
-      switch (question.type) {
+      switch (data.type) {
         case "info":
-          const { data: infoRequestArray, error: infoRequestError } = await tc(
+          const infoRequest = await databaseInteraction(
             db
               .select()
               .from(questionInfo)
-              .where(eq(questionInfo.id, question.questionId!)),
+              .where(eq(questionInfo.id, data.questionId!)),
           );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)